sdk/template/Azure.Template/samples/Sample1.HelloWorldAsync.md

```diff
@@ -0,0 +1,25 @@
+# Get secret
```

Comment:
```suggestion
# Get secret asynchronously
```

Reply:
Also, why does this file use a period as a separator in the file name while Sample1_HelloWorld.md does not? I'd stick with an underscore for both as we normally do throughout the repo (mostly).